php-checks/src/main/resources/org/sonar/l10n/php/rules/php/Drupal_profile.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@ivandalbosco I don't agree. It's better to remove this file with awareness each time, than to always have this file somewhere in the plugin without awareness.